java/runtime/src/main/java/org/ray/runtime/task/ArgumentsBuilder.java

```diff
@@ -22,10 +23,13 @@
    */
   private static final int LARGEST_SIZE_PASS_BY_VALUE = 100 * 1024;
 
+  private static final NativeRayObject PYTHON_DUMMY_TYPE = ObjectSerializer
+      .serialize("__RAY_DUMMY__".getBytes());
```

Comment:
Can you add a comment here and in `signature.py`, explaining that both need to be sync'ed?